src/readii_2_roqc/visualization/plot_prediction.py

```
@@ -1,20 +1,25 @@
 import matplotlib.pyplot as plt
 import pandas as pd
 import seaborn as sns
+
 from damply import dirs
 from matplotlib.figure import Figure
-from readii.io.loaders.general import loadImageDatasetConfig
 from readii.io.writers.plot_writer import PlotWriter
 from readii.utils import logger
+from readii_2_roqc.utils.loaders import DATA_SPLIT_CHOICES, load_dataset_config
+
+logger = logging.getLogger(__name__)  
```

Ruff (0.14.1)

Redefinition of unused `logger` from line 12

(F811)